docs/deployment/getting-started-locally/manage-s3-emulator.md

```diff
@@ -0,0 +1,69 @@
+# Manage S3 Emulator
+
+This document is a step-by-step guide on how to configure, deploy and cleanup [LocalStack](https://localstack.cloud/), within a [kind](https://kind.sigs.k8s.io/) cluster. LocalStack emulates AWS services locally, which allows the `Etcd` cluster to interact with AWS S3. This setup is ideal for local development and testing.
+
+## 00-Prerequisites
+
+Ensure that you have setup the development environment as per the [documentation](../prepare-dev-environment.md).
+
+> **Note:** It is assumed that you have already created kind cluster and the `KUBECONFIG` is pointing to this Kubernetes cluster.
+
+### Installing AWS CLI
+
+To interact with `LocalStack` you must also install the AWS CLI `(version >=1.29.0 or version >=2.13.0)`
+On macOS run:
+
+```bash
+> brew install awscli
+```
+
+For other OS, please check the [AWS CLI installation documentation](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+## 01-Deploy LocalStack
+
+```bash
+> make deploy-localstack
+```
+
+The above make target will deploy `LocalStack` in the target Kubernetes cluster.
+
+## 02-Setup S3 Bucket
+
+Configure `AWS CLI` to interact with LocalStack by setting the necessary environment variables. This configuration redirects S3 commands to the LocalStack endpoint and provides the required credentials for authentication.
+
+```bash
+> export AWS_ENDPOINT_URL_S3="http://localhost:4566"
```

Comment:
endpoint is usually used by S3 compatible providers not by S3 itself.
why not add this link: https://github.com/gardener/etcd-backup-restore/tree/master/example/storage-provider-secrets

Reply:
I did not quite understand this comment. This was an existing document which i have just moved to another place. Also these are instructions on setting up a local s3 compatible bucket.

Reply:
yes, my bad ... I missed it's writtern only for s3 emulator.